src/pages/gateway/create-mesh.md

```diff
@@ -36,7 +36,7 @@ Workspaces that already have an API mesh will have "API Mesh" displayed on their
 
 When creating or updating a mesh, the file to upload must have the `.json` filename extension.
 
-1. Create and save a JSON configuration file that defines the properties of your mesh. Your mesh is defined by a combination of [handlers] and [transforms]. In this example, the file name is `mesh.json`.
+1. Create and save a JSON configuration file that defines the properties of your mesh. Your mesh is defined by a combination of [handlers] and [transforms]. In this example, the filename is `mesh.json`.
```

Comment:
These changes appear to be incorrect: https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/f/file-name

Reply:
It's correct according to the Adobe Glossary